File: python/different.py
#!/usr/bin/env python
""" An attempt to solve "A Different Problem" on Kattis """

import sys

def main():
    data = [line.rstrip() for line in sys.stdin.readlines()]
    for line in data:
        first, second = line.split(" ")
        print("{}".format(abs(int(first) - int(second))))

if __name__ == "__main__":
    main()

File: src/bin/different.rs
use std::io::{self, BufRead};

fn main() {
    // set things up
    let stdin = io::stdin();

    for raw_line in stdin.lock().lines() {
        let line = raw_line.unwrap();
        let mut split = line.split(' ');
        let first = split.next().unwrap().parse::<isize>().unwrap();
        let second = split.next().unwrap().parse::<isize>().unwrap();
        let diff = first - second;
        println!("{}", diff.abs());
    }
}
